students/Richard.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Richard</title>
+</head>
+<body>
+    <img src="http://www.dionisioarte.com.br/wp-content/uploads/2016/11/josan-gonzalez-sci-fi-cyberpunk-ilustracao-12.gif" width="783" height="75"/>
+   <h1>What's up everyone!</h1>
+     <p>
+         My name is <b>Richard</b> and I'm looking forward to becoming a great programmer one day.
+         I have been studying C#, SQL, ASP .NET Core, and Unity.
+     </p>
+    <p>
+        These are my professional goals:
+    </p>
+    <ol>
+        <li>Web Developer</li>
+        <li><a href="https://dronerush.com/best-drone-apps-7629/">Drone Tech</a></li>
+        <li>VR Developer</li>
+    </ol>
+    <p>
+        When I'm not programming I love to indulge into my other passions as well. 
+        Such as:
+    </p>
+    <ul>
+        <li>Socializing in VR social spaces</li>
+        <li>Flying my drone</li>
+        <li>Satisfying my nostalgia for my favorite genres from the 80s and 90s(action movies, anime and Southern rap)</li>
+    </ul>
+    <p>
+        I see a trajectory that VR will be the next computational frontier to explore and innovate. You might as<br />well come join the party
+        now. Even if you dont have a headset you can get a glimpse of what to expect in the browser as well. If you want<br />to meet up and talk 
+        and explore virtual worlds we can hang out in<a href="https://www.vrchat.com/">VRChat.</a> 

```

Comment:
🔧 Typo (missing space between "in" and "VRChat"... screen shot below)

<img width="109" alt="image" src="https://user-images.githubusercontent.com/1994207/89739729-23707480-da51-11ea-80cc-420bbb4b013c.png">